src/main/java/uk/gov/hmcts/divorce/common/event/ApplyForFinalOrder.java

```diff
@@ -75,13 +75,13 @@ private PageBuilder addEventConfig(ConfigBuilder<CaseData, State, UserRole> conf
         log.info("Apply for Final Order about to submit callback invoked");
 
         CaseData data = details.getData();
-
-        if (data.getApplicationType().isSole()) {
-            notificationDispatcher.send(soleAppliedForFinalOrderNotification, data, details.getId());
-        }
-
         State endState = details.getState();
+
         if (details.getState() == AwaitingFinalOrder) {
```

Comment:
@nitinprabhu I haven't combined them because the state should change to FinalOrderRequested for joint applications if the state is AwaitingFinalOrder too